cellprofiler/gui/help/content.py

```diff
@@ -436,8 +436,13 @@ def __image_resource(filename):
    -  A *progress bar* which gives the elapsed time and estimates the
       time remaining to process the full image set.
 
-   At the end of each cycle, CellProfiler saves the measurements in the
+   At the end of each cycle:
+   -  If you are creating a MATLAB or HDF5 output file, CellProfiler saves the measurements in the
    output file.
+   -  If you are using the **ExportToDatabase** module, CellProfiler saves the measurements in the
+   output database.
```

Comment:
I think this formatting is incorrect. It should be 

```
-  If you are using...
    output database
```

(Same with the line below.)